packages/apps/crm/src/components/organisms/DropdownEventView/DropdownEventView.tsx

```diff
@@ -21,7 +21,7 @@ import {StyleSheet, View} from 'react-native';
 import {useNavigation, useTranslator} from '@axelor/aos-mobile-core';
 import {EventCard} from '../../molecules';
 import {Text} from '@axelor/aos-mobile-ui';
-import {getLastEvent, getNextEvent} from '../../../utils/dateEvent';
```

Comment:
Do not change this here please.
You can create a generic helper if you want but make it flexible by leaving fields name definition in arguments and then call this helper in dateEvent utils.